### DISCUSSION TITLE
production https://example:8044

### DISCUSSION THREAD

**Discussion by kgalanos** [filamentphp/filament#13256] [Help]
### Package

Panel builder

### Package Version

v3.2

### How can we help you?

Hello
I develop an app with filament with url https://example.test
When I go to production we setup nginx with no default port so the app listen https://example:8044
Show the first (login page) but after login redirect to https://example (without port)
If I put manualy the port works. but all others link to panel resources, widgets, pages is without port I mean the link is
https://example and not the correct https://example:8044.
I try in .env
APP_URL not work
APP_PORT not work
ASSET_URL works !! because at start no css, js was correct
now i am confuse how can I fix it in production

**Comment by christmex** [comment]
hi @kgalanos , did you set your ``` APP_URL=https://example:8044 ``` ?

or maybe that is something you need to set via .htaccess?

maybe you can share the console log, network tab, or event ss how does it look

Thanks.
